Sources/SQL/SQLExpression.swift

```diff
@@ -240,6 +240,11 @@ public indirect enum GenericSQLExpression<Literal, Bind, ColumnIdentifier, Binar
     /// See `SQLExpression`.
     case _coalesce([`Self`])
     
+    /// See `SQLLiteral`.
+    public init(rawSQL value: String) {
```

Comment:
I think `raw:` as a label would suffice. There is already `SQL` in the type name.

Reply:
I would prefer to keep the `SQL` in here. Otherwise, it's not clear whether this could be raw data, a raw string, or something else. I suggest erring on the safe side of avoiding any potential confusion.

Reply:
This should be added as an extension to `SQLExpression` anyway. And should be a static method to match the other conveniences. I think `static func raw(_ sql: String)` would be best.